framework/Metrics/Metric.py

```diff
@@ -65,13 +71,56 @@ def _readMoreXML(self,xmlNode):
     """
     self._localReadMoreXML(xmlNode)
 
+  def evaluate(self, x, y, weights = None, axis = 0, **kwargs):
+    """
+      This method compute the metric between x and y
+      @ In, x, numpy.array or instance of Distributions.Distribution, array containing data of x,
+        or given distribution.
+      @ In, y, numpy.array, or instance of Distributions.Distribution, array containing data of y,
+        or given distribution.
+      @ In, weights, numpy.array, an array of weights associated with x
```

Comment:
you did not fix this docstring in all the code

Reply:
fixed now.